examples/helpers/camera_controller.rs

```diff
@@ -125,7 +125,7 @@ Freecam Controls:
 }
 
 fn run_camera_controller(
-    time: Res<Time>,
+    time: Res<Time<Real>>,
```

Comment:
What was the rationale here? Don't we generally want to use the default (virtual) time for things that move in Update?